src/components/PantallaValidacion/index.module.scss

```
@@ -0,0 +1,26 @@
+.PantallaValidacion{
```

Cuidado que acá  (y en algunas clases más) está chanfleado el camelCase
```suggestion
.pantallaValidacion {
```